web/src/attribute-filters.tsx

```diff
@@ -273,6 +273,11 @@ export const getNamespaceMatcher = (namespace?: string): LabelMatcher | undefine
   };
 };
 
+const isValueARegex = (value: string) => {
+  const testRegex = /\\|\[|\{|\(|\)|\^|\$|\.|\*|\?|\||\+|\/|\]/;
+  return testRegex.test(value);
```

Comment:
This looks like it will return true for a string that contains any one of the special characters (`\`, `[`, `{`, `(`, `)`, `^`, `$`, `.`, `*`, `?`, `|`, `+`, `/`, `]`), but don't we need to support exact matching against string values that contain these characters?

Reply:
In this case we are only matching [k8s names](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names) which exclude these characters as they are DNS names. The only exception is dot `'.'`. I adjusted the code to consider this case and added some tests and comments for clarity.

Reply:
Thanks for the explanation!

If we use a character class (square brackets), I think only the `\` and `]` need to be escaped. So like `[{}()[\]^$*+?|/\\]|\.[*+?]`, which I think is at least a bit easier to read.

Note that I also added the `}` character in that regex, which I guess should be included for completeness.

Reply:
Thnx I changed the regex with your suggestion and is more readable now